constraint/config/crds/templates.gatekeeper.sh_constrainttemplates.yaml

```diff
@@ -19,18 +19,13 @@ spec:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: ConstraintTemplate is the Schema for the constrainttemplates
-          API
+        description: ConstraintTemplate is the Schema for the constrainttemplates API
```

Comment:
@julianKatz this is the same line break I'm seeing as well

Reply:
I wonder if this is version skew against the version of controller-gen used. Did we dockerize that here?

Reply:
Or maybe kustomize

Reply:
> unit test failed:
> 
> > package sigs.k8s.io/controller-tools/cmd/controller-gen@v0.5.0: can only use path@version syntax with 'go get'
> > make: *** [Makefile:34: gen-dependencies] Error 1

Ah, I found it.